### ISSUE TITLE
feat: add a way of resuming training

### ISSUE THREAD

**Issue by pedrohbtp** [p-christ/Deep-Reinforcement-Learning-Algorithms-with-PyTorch#23]
First of all, great work with the library!

My understanding of the library is that it starts training from scratch every time.
Would there be an easy way of making possible to resume training for an agent?

I would be glad to try to help if you could give some guidance. 

Thanks!

**Comment by alex1ai** [comment]
You could write a save/load method for the network parameters in the Base_Agent. As actor-critic/q/gradient methods have different (number of) networks you could filter the Base_Agent.__dict__ class for nn.Module instances and save those to file/ load them to the agent. 
At least this is how I implemented it to load different agents for inference
